drivers/mrf24j40/mrf24j40_netdev.c

```diff
@@ -330,6 +330,20 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             }
             break;
 
+#ifdef MODULE_NETDEV_IEEE802154_OQPSK
+
+        case NETOPT_IEEE802154_PHY:
+            assert(max_len >= sizeof(int8_t));
+            *(uint8_t *)val = IEEE802154_PHY_OQPSK;
+            return sizeof(uint8_t);
+
+        case NETOPT_OQPSK_RATE:
```

Comment:
Is this netopt now being used to activate the proprietary mode in radios? From the user perspective wouldn't it be misleading that, if using different radios, when this is set communication does not longer work?

Reply:
802.15.4-OQPSK only species 250 kbit/s but many radios implement higher modes. So this netopt is *only* used to activate the proprietary modes.

`NETOPT_MR_OQPSK_RATE` is used to set standard-compliant 802.15.4g MR-OQPSK rates.

Reply:
I see. Ok then I'm fine with this, but I think that a follow-up PR which improves the documentation of `NETOPT_OQPSK_RATE` would be good.